goatshriek/stumpless
# Implement perror

Implemented PERROR option for stumpless. Defined function `stumpless_entry_to_errorstream` in file `src/target.c` on `line 782`
and defined the option `STUMPLESS_OPTION_PERROR` in file `include/stumpless/option.h`. Changes done in `stumpless_add_entry` function in file `src/target.c`. Added test case in `test/function/target.cpp.

Steps to test : 
`cmake --rebuild .`
`cmake --build . --target check`
`./function-test-target`

Check WithPerror category, a log is printed on the terminal with timestamp and message test message with perror
Attached is the image of the output
![image](https://user-images.githubusercontent.com/56515325/146572772-0848e9db-6e35-473c-afc1-2b24c963dc59.png)
